### PR TITLE
Fix management snippet

### PR DESCRIPTION
**Pull request by kwart**
This PR fixes failing build by adding a new-line in a log snippet (in 3rd commit).
There are 2 other commits which are just cosmetic:
* gitignore update
* removing trailing spaces in `management.adoc`

### CODE REVIEW

**Comment by kwart** [comment]
I'm also trying a PR builder job now: https://hazelcast-l337.ci.cloudbees.com/job/reference-manual-pr-builder/

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,8 @@ build
 static
 vendor
 out/
+# Eclise IDE
```
Comment (Serdaro) [inline .gitignore]: Just to make it look good. Eclipse :)

**Comment by kwart** [comment]
run-lab-run

**Comment by kwart** [comment]
Thanks, Serdar. The typo is fixed now.

**Pull request merged**